src/clients/Lighthouse.ts

```diff
@@ -0,0 +1,83 @@
+import { AppClient, IOContext, InstanceOptions } from '@vtex/api'
+import { stringify } from 'querystring'
+
+const LH_TIMEOUT_MS = 1e5
```

Comment:
_nit_: Just to easily see how many seconds it is
```suggestion
const LH_TIMEOUT_MS = 100 * 1000
```
I'm pretty sure that router max timeout is 55s, so maybe we want less timeout here

Reply:
Maybe add this as a `private static readonly` property on `Lighthouse`?
```ts
export class Lighthouse extends AppClient {
  private static readonly TIMEOUT_MS = 100 * 1000
```

Reply:
Got it! You are right, for this toolbet command, router will limit the timeout. The lightouse app usually runs on some event listeners and the route dont limit the execution, because some lighthouse audits can take more than 55s to finish (Just explaning why this values was so high).